test/Tabs.spec.js

```diff
@@ -1,10 +1,7 @@
 /* eslint react/no-multi-comp: 0, react/prop-types: 0 */
 import React from 'react';
-import ReactDOM from 'react-dom';
-
 import { mount } from 'enzyme';
-import { Nav, NavItem, NavLink, TabContent, TabPane } from 'reactstrap';
```

Comment:
ah, totally missed the unused deps in these tests. Currently eslint only lints the main src directory. Thanks!